.circleci/config.yml

```diff
@@ -168,6 +121,6 @@ workflows:
       - deploy-doc:
           requires:
             - build-doc
-          filters:
-            branches:
-              only: main  # don't deploy to GitHub pages from other branches
+          #filters:
```

Comment:
This worries me. The filter seems necessary

Reply:
I'll put it back, but the script skips the deploy if it's a wrong branch anyway.